teacher.rb

```diff
@@ -0,0 +1,16 @@
+module TeacherModule
+  require_relative 'person'
+  include PersonModule
+  class Teacher < Person
+    attr_accessor :specialization
+
+    def initialize(specialization)
+      @specialization = specialization
+      super
+    end
```

Comment:
- [ ] Please also make the` attr_accessor :specialization` abode the class student. so it will be accessible before starting the definition.
- [ ] Also please make the definition take the necessary argument ( age, classroom, name = 'Uknown', parent_permision = true)
- [ ] also the super should take the (age, name, parent-permission = true)